ztp/gitops-subscriptions/argocd/deployment/talm-operator-subscription.yaml

```diff
@@ -0,0 +1,14 @@
+# Subscription CR to install Topology Aware Lifecycle Manager (TALM)
```

Comment:
I think this needs to be included in the kustomization.yaml or it won't get installed based on the user documentation (says to run oc apply -k <this directory>)